### PR TITLE
libspiro, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Test fails 1 out of 18 (multi thead test):
```
Multi-thread testing of libspiro.
running simultaneous threads[0..499]
bad pthread_join[465]
...
```

**Comment by korli** [comment]
is it used by fontforge?

**Comment by Begasus** [comment]
https://github.com/haikuports/haikuports/pull/3819#issuecomment-488330481

> is it used by fontforge?

File: media-libs/libspiro/libspiro-0.5_20180219~git.recipe [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+X,Y constraints.
+
+The main user(s) of Libspiro are Font editing programs such as FontForge, and \
+forked copies have been seen in more artistic programs such as InkScape, or \
+in libraries such as GEGL. There may be other possible uses in future, and \
+this library has strong potential in graphical and vector type programs."
+HOMEPAGE="https://homepage/of/projectx.org"
+COPYRIGHT="2007 Raph Levien
+	George Williams"
+LICENSE="MIT
```
Comment (korli) [inline media-libs/libspiro/libspiro-0.5_20180219~git.recipe]: Why MIT?

Comment (Begasus) [reply media-libs/libspiro/libspiro-0.5_20180219~git.recipe]: Wasn't sure about this :)

File: media-libs/libspiro/libspiro-0.5_20180219~git.recipe [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+The main user(s) of Libspiro are Font editing programs such as FontForge, and \
+forked copies have been seen in more artistic programs such as InkScape, or \
+in libraries such as GEGL. There may be other possible uses in future, and \
+this library has strong potential in graphical and vector type programs."
+HOMEPAGE="https://homepage/of/projectx.org"
+COPYRIGHT="2007 Raph Levien
+	George Williams"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="8c7a31eda7dbd097fd0fae19ca7c7a67bc489d02"
```
Comment (korli) [inline media-libs/libspiro/libspiro-0.5_20180219~git.recipe]: Why this git revision?

Comment (korli) [reply media-libs/libspiro/libspiro-0.5_20180219~git.recipe]: latest rev, nevermind

**Comment by korli** [comment]
Looks good to me.

File: media-libs/libspiro/libspiro-0.5_20180219~git.recipe [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+INSTALL()
+{
+	make install
+
+	# remove libtool file
+	rm $libDir/*.la
+
+	prepareInstalledDevelLib libspiro
+	fixPkgconfig
```
Comment (korli) [inline media-libs/libspiro/libspiro-0.5_20180219~git.recipe]: is libspiro.pc actually installed? the arch recipe seems to copy the file manually.

Comment (Begasus) [reply media-libs/libspiro/libspiro-0.5_20180219~git.recipe]: It is :)
```
~> pkgconf-x86 libspiro --cflags
-I/packages/libspiro_x86-0.5_20180219~git-1/.self/develop/headers/x86 
~> pkgconf-x86 libspiro --libs
-L/packages/libspiro_x86-0.5_20180219~git-1/.self/develop/lib/x86 -lspiro
```

**Pull request merged**